### ISSUE TITLE
Docker CLI - Custom Terminal app

### ISSUE THREAD

**Issue by nolsen311** [kitematic/kitematic#905]
Would be nice to select which Terminal app is used when loading the Docker CLI. My PATH isn't correctly set up in Terminal.app, I use ITerm2. Anyway, some kind of workaround would be nice.

**Comment by mchiang0610** [comment]
Thanks @nolsen311 for submitting. We will review this.

**Comment by FrenchBen** [comment]
@nolsen311 I use iterm2 also and that's what Kitematic launches every time. 
What version of iterm2 are you running?

**Comment by nolsen311** [comment]
Whatever the most current is... 2.9.20150626

--Nate Olsen

**Comment by FrenchBen** [comment]
@nolsen311 What was the output of the `osascript` command?

**Comment by nolsen311** [comment]
My Iterm is also running as a pull-down more-or-less constantly.

--Nate Olsen

**Comment by Chekote** [comment]
The command used to detect iTerm in resources/terminal only works for iTerm1, not iTerm2. My system has iTerm2 installed, and I get the following output:

```shell
$ osascript -e 'exists application "iTerm"'
0:26: execution error: An error of type -10665 has occurred. (-10665)
$ osascript -e 'exists application "iTerm2"'
true
```

So my current workaround is to change iTerm to iTerm2 in resources/terminal.

**Comment by FrenchBen** [comment]
:+1: @Chekote thanks for confirming - @nolsen311 would you mind closing this thread as we have a pending PR to fix this whenever the 2.9+ is released as stable:
https://github.com/kitematic/kitematic/pull/855
